templates/login_page.html

```diff
@@ -67,6 +67,9 @@ <h1> Vacation Planner </h1>
             <button type="submit" class="btn btn-primary login-btn btn-block">Sign in</button>
             <!-- TODO: Add Login buttons for Social media (FB, G, TWT) as a separate PR -->
         </form>
+        <div class="mt-1">
+            <p><a href="/v1/forgot-password">forgot password?</a></p>
```

Comment:
maybe we can make "forgot-password" also a button. For example, we can use "Login with Google" as a reference?

Reply:
"forgot-password" is typically a link on most websites, I am adjusting the link to align with the sign in button.